backend/tests/Feature/PublicationTest.php

```diff
@@ -64,22 +61,19 @@ public function publicationMutationProvider(): array
      * @dataProvider publicationMutationProvider
      * @return void
```

Comment:
```suggestion
     * @param mixed $data
     * @param mixed $expectedData
     * @return void
```
I encourage adding comments for these parameters as well.